app/models/miq_server/log_management.rb

```diff
@@ -124,13 +124,14 @@ def last_log_sync_message
   def post_logs(options)
     taskid = options[:taskid]
     task = MiqTask.find(taskid)
+    context = options[:context]
 
     # the current queue item and task must be errored out on exceptions so re-raise any caught errors
-    raise "Log depot settings not configured" unless log_depot
-    log_depot.update_attributes(:support_case => options[:support_case].presence)
+    raise "Log depot settings not configured" unless log_depot(context)
+    log_depot(context).update_attributes(:support_case => options[:support_case].presence)
```

Comment:
Yes, we still need this downstream, right @bdunne.  Maybe it can be converted to a hook to be used downstream or by users. I can imagine, it might be useful elsewhere by MIQ upstream users to specify something like this to uniquely identify a set of logs.